hawkbit-dmf/hawkbit-dmf-rabbitmq-test/src/main/java/org/eclipse/hawkbit/rabbitmq/test/RabbitMqSetupService.java

```diff
@@ -77,7 +78,7 @@ public ConnectionFactory newVirtualHostWithConnectionFactory() {
 
     @PreDestroy
     public void deleteVirtualHost() {
-        if (StringUtils.isEmpty(virtualHost)) {
+        if (StringUtils.hasText(virtualHost)) {
```

Comment:
Shouldn't it be !StringUtils.hasText here ? 
isEmpty <-> !hasText

Reply:
yes, amended the commit